tests/test_tokenization_clip.py

```diff
@@ -30,18 +30,20 @@ class CLIPTokenizationTest(TokenizerTesterMixin, unittest.TestCase):
 
     tokenizer_class = CLIPTokenizer
     rust_tokenizer_class = CLIPTokenizerFast
-    test_rust_tokenizer = False
-    from_pretrained_kwargs = {"add_prefix_space": True}
+    test_rust_tokenizer = True
+    from_pretrained_kwargs = {}
     test_seq2seq = False
 
     def setUp(self):
         super().setUp()
+        # temporary addition: to test the new slow to fast converter
+        self.tokenizers_list = [(CLIPTokenizerFast, "SaulLu/clip-vit-base-patch32", {})]
```

Comment:
This is a hack to run the tests on the new version of the tokenizer fast.

If we merge this PR, we have to update the fast files on the `openai/clip-vit-base-patch32` repo and remove this line.